### PR TITLE
Add examine text for when someone is using VR goggles or other

### PR DESCRIPTION
**Pull request by FlameArrow57**
[PLAYER ACTIONS][QoL]

## About the PR 
This PR just adds a unique examine text for people that appears if they are using something like VR or scuttlebot goggles.

It will appear instead of the "appears to be staring blankly into space." message if they are still in the game

image

## Why's this needed? 
Prevents confusion from if someone actually quit the game or not

## Changelog 

```changelog
(u)FlameArrow57
(+)Players using VR goggles or a scuttlebot controller will now have an examine text saying their mind is "elsewhere."
```

### CODE REVIEW

**Comment by RelentlessGarbage** [comment]
This is a great, straightforward change. If it reflects whether or not the players are actually active in VR (i.e. it'll still say that they're staring blankly into space if they're inactive with VR goggles on), then I'd like to see it merged.

**Comment by FlameArrow57** [comment]
Yup, it does that too

File: code/mob/living/carbon/human/procs/get_desc.dm [hunk 1]
```diff
@@ -346,7 +346,11 @@
 				. += "<br><span class='alert'>[src.name] has a blank expression on [his_or_her(src)] face.</span>"
 
 			if (!src.client && !src.ai_active)
-				. += "<br>[src.name] seems to be staring blankly into space."
+				if (find_player(src.last_ckey)?.client) // using vr goggles or something similar
```
Comment (TobleroneSwordfish) [inline code/mob/living/carbon/human/procs/get_desc.dm:349]: Won't this catch stuff like being respawned or mindswapped into a different body as well? Probably best to specifically check if they are in VR or using a scuttlebot (or have those things set a flag on the old body so we can tell)

Comment (FlameArrow57) [reply code/mob/living/carbon/human/procs/get_desc.dm]: Yeah I think so, fixed

Comment (FlameArrow57) [reply code/mob/living/carbon/human/procs/get_desc.dm]: Though it won't work for if you mindswap into a scuttlebot while controlling it. That's a pretty rare case though, maybe could be considered a "stuck in vr" type feature?

File: code/mob/living/carbon/human/procs/get_desc.dm [hunk 2]
```diff
@@ -346,7 +346,21 @@
 				. += "<br><span class='alert'>[src.name] has a blank expression on [his_or_her(src)] face.</span>"
 
 			if (!src.client && !src.ai_active)
-				. += "<br>[src.name] seems to be staring blankly into space."
+				var/using_vr_goggles = FALSE
+				var/mob = find_player(src.last_ckey)?.client?.mob
+
+				if (istype(mob, /mob/living/carbon/human/virtual))
+					using_vr_goggles = TRUE
```
Comment (TobleroneSwordfish) [inline code/mob/living/carbon/human/procs/get_desc.dm:353]:
```suggestion
					var/mob/living/carbon/human/virtual/vr_person = mob
					if (!vr_person.isghost)
						using_vr_goggles = TRUE
```

Comment (FlameArrow57) [reply code/mob/living/carbon/human/procs/get_desc.dm:353]: I guess which case would this fall under? You can only be an observer if you've died, in which case the dead message would appear first, or are an admin, release your body, and then go into vr space which I don't think that is going to occur very often

Comment (TobleroneSwordfish) [reply code/mob/living/carbon/human/procs/get_desc.dm:353]: Right, but this will eliminate those corner cases so why not add it?

Comment (FlameArrow57) [reply code/mob/living/carbon/human/procs/get_desc.dm:353]: Sure

**Pull request merged**